config/wikis/arwiki.yaml

```diff
@@ -12,14 +14,5 @@ models:
     reverted:
         tuning_params:
             max_depth: 5
-
-host: ar.wikipedia.org
-trusted_groups:
-    - sysop
-    - oversight
-    - editor
-    - bot
-    - rollbacker
-    - checkuser
-    - abusefilter
-    - bureaucrat
+        pop_rate_true: 0.035186595582635184
```

Comment:
:) and we can draw from a proper data source once this work is ready...

Reply:
Yes, that's part of my evil plan